package.json

```diff
@@ -67,29 +67,31 @@
   },
   "homepage": "https://github.com/react-native-community/datetimepicker#readme",
   "devDependencies": {
+    "@callstack/react-native-visionos": "^0.73.0",
     "@react-native-segmented-control/segmented-control": "^2.4.0",
     "@react-native/eslint-config": "^0.72.2",
-    "@react-native/metro-config": "^0.73.2",
+    "@react-native/metro-config": "^0.73.0",
+    "@rnx-kit/metro-config": "^1.3.15",
     "@semantic-release/git": "^10.0.1",
     "@testing-library/react-native": "9.1.0",
     "babel-jest": "^29.5.0",
     "detox": "^20.19.3",
     "eslint": "^8.42.0",
     "eslint-plugin-ft-flow": "^2.0.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "flow-bin": "^0.182.0",
-    "flow-typed": "^3.8.0",
+    "flow-bin": "^0.217.0",
+    "flow-typed": "^3.9.0",
     "jest": "^29.5.0",
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.41",
     "patch-package": "^6.4.7",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^2.8.8",
     "react": "18.2.0",
-    "react-native": "^0.72.7",
+    "react-native": "^0.73.0",
```

Comment:
Could we update to the latest RN 74 RC, so that bridgeless mode can also be tested?
Thank you

Reply:
Could we not test bridgeless with 0.73? I think I'd be willing to do that after I can fix the CI for the current changes 😅 (Right now just New arch + Android fails)

Reply:
okay, let's do it separately. Keep this with 0.73 and I'll handle bridgeless with 0.74 separately 👍